pkg/tenant/db/members.go

```diff
@@ -22,29 +22,35 @@ const (
 )
 
 type Member struct {
-	OrgID        ulid.ULID    `msgpack:"org_id"`
-	ID           ulid.ULID    `msgpack:"id"`
-	Email        string       `msgpack:"email"`
-	Name         string       `msgpack:"name"`
-	Role         string       `msgpack:"role"`
-	Status       MemberStatus `msgpack:"status"`
-	Created      time.Time    `msgpack:"created"`
-	Modified     time.Time    `msgpack:"modified"`
-	DateAdded    time.Time    `msgpack:"date_added"`
-	LastActivity time.Time    `msgpack:"last_activity"`
+	OrgID        ulid.ULID `msgpack:"org_id"`
+	ID           ulid.ULID `msgpack:"id"`
+	Email        string    `msgpack:"email"`
+	Name         string    `msgpack:"name"`
+	OrgName      string    `msgpack:"org_name"`
+	OrgDomain    string    `msgpack:"org_domain"`
+	UsageType    string    `msgpack:"usage_type"`
+	Interests    []string  `msgpack:"interests"`
+	Role         string    `msgpack:"role"`
+	Invited      bool      `msgpack:"invited"`
+	JoinedAt     time.Time `msgpack:"joined_at"`
+	LastActivity time.Time `msgpack:"last_activity"`
+	Created      time.Time `msgpack:"created"`
+	Modified     time.Time `msgpack:"modified"`
 	gravatar     string
 }
 
 type MemberStatus uint8
 
 const (
 	MemberStatusPending MemberStatus = iota
-	MemberStatusConfirmed
+	MemberStatusOnboarding
+	MemberStatusActive
```

Comment:
The major changes here is adding a state for the onboarding and making it a derived status, the status is mostly used for the frontend but there is one place that we're checking it on the backend that I've noted below.